learn-spam.sieve

```diff
@@ -1,8 +1,4 @@
 require ["vnd.dovecot.pipe", "copy", "imapsieve", "environment", "variables", "imap4flags"];
 
-if environment :matches "imap.user" "*" {
```

Comment:
why do you remove the username part?

Reply:
the full commit 2cdf702403e084ef56bb4369d738eb166565bf24 should explain all this.